app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :meme
```

Comment:
You'll have to check Rails syntax for this, but right now if I delete a Meme, the Comments become meme-less but don't get deleted (so they're in an invalid state). There's a way to set the relationship as cascading on delete?